iron-multi-selectable.html

```diff
@@ -106,6 +116,10 @@
         for (var i = 0; i < selectedItems.length; i++) {
           this._selection.setItemSelected(selectedItems[i], true);
         }
+        // Check for items, since this array is populated only when attached
+        if (this.fallbackSelection && this.items.length && !this._selection.get().length) {
+          this.selectedValues = this._fallbackArray;
```

Comment:
The `_fallbackArray` is meant to be private/internal implementation. Modifying would be at the risk of the user imo.